addons/account/models/account_account.py

```diff
@@ -95,6 +95,10 @@ def _check_user_type_id_unique_current_year_earning(self):
         ('code_company_uniq', 'unique (code,company_id)', 'The code of the account must be unique per company !')
     ]
 
+    non_trade = fields.Boolean(default=False,
+                               help="If set, this account will belong to Non Trade Receivable/Payable in reports and filters.\n"
+                                    "If not, this account will belong to Trade Receivable/Payable in reports and filters.")
```

Comment:
It's a personal opinion but don't you think writing it like
```py
non_trade = fields.Boolean(
    default=False,
    help="If set, this account will belong to Non Trade Receivable/Payable in reports and filters.\n"
         "If not, this account will belong to Trade Receivable/Payable in reports and filters.",
)
```
has a more effective space managment (no more unused empty space on the left)?
If you don't agree, ignore this comment

Reply:
Personal taste indeed, I like the variable name being isolated from its content, I think it's clearer. If it is important to you I can make an exception :)